src/components/layout/addons/AddonItem.js

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import humanFormat from 'human-format';
+import { styles, animation, Cardinal, AvatarList, Icon } from '@storybook/design-system';
+import customSVG from '../../../images/addon-catalog/custom.svg';
+
+const { hoverEffect, spacing, color, typography, breakpoint } = styles;
+const { inlineGlow } = animation;
+
+const VerifiedBadge = styled(Icon).attrs({ icon: 'verified', block: true })`
+  color: ${(props) => (props.appearance === 'official' ? color.secondary : color.green)};
+  width: 14px;
+  height: 14px;
+  margin-left: ${spacing.padding.small}px;
+  margin-bottom: 2px;
+`;
+VerifiedBadge.propTypes = {
+  appearance: PropTypes.oneOf(['official', 'integrator']),
+};
```

Comment:
Is there a benefit to adding 0 to a styled component, especially if they aren't required props?

Reply:
imo you get the same benefit as any other component. Was just doing it out of habit but, happy to adopt whatever convention we use in the codebase.

Reply:
Makes sense, though with other components when you have non-required props, you have to specify a default typically -- that's what I was getting at.

Reply:
Good point, these should actually be required